### ISSUE TITLE
FATAL Error

### ISSUE THREAD

**Issue by Blunder-G** [Alex-the-666/Ice_and_Fire#4151]
My performance decreases and MY COMPUTER crashes after this console message:

```
[14:58:53] [Render thread/FATAL]: Error executing task on Client
java.lang.NullPointerException: null
	at com.github.alexthe666.citadel.server.entity.datatracker.EntityProperties.getEntity(EntityProperties.java:63) ~[citadel:1.7.3-1.16.5]
	at com.github.alexthe666.iceandfire.entity.props.ChainEntityProperties.loadNBTData(ChainEntityProperties.java:54) ~[iceandfire:2.1.8-1.16.5]
	at com.github.alexthe666.citadel.server.entity.datatracker.EntityProperties.loadTrackingSensitiveData(EntityProperties.java:112) ~[citadel:1.7.3-1.16.5]
	at com.github.alexthe666.citadel.ClientProxy.handlePropertiesPacket(ClientProxy.java:128) ~[citadel:1.7.3-1.16.5]
	at com.github.alexthe666.citadel.server.message.PropertiesMessage$Handler.lambda$handle$0(PropertiesMessage.java:50) ~[citadel:1.7.3-1.16.5]
	at net.minecraftforge.fml.network.NetworkEvent$Context.enqueueWork(NetworkEvent.java:215) ~[forge:?]
	at com.github.alexthe666.citadel.server.message.PropertiesMessage$Handler.handle(PropertiesMessage.java:48) ~[citadel:1.7.3-1.16.5]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.lambda$tryDecode$3(IndexedMessageCodec.java:128) ~[forge:?]
	at java.util.Optional.ifPresent(Unknown Source) ~[?:1.8.0_291]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.tryDecode(IndexedMessageCodec.java:128) ~[forge:?]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.consume(IndexedMessageCodec.java:162) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.networkEventListener(SimpleChannel.java:80) ~[forge:?]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.fml.network.NetworkInstance.dispatch(NetworkInstance.java:86) ~[forge:?]
	at net.minecraftforge.fml.network.NetworkHooks.lambda$onCustomPayload$1(NetworkHooks.java:91) ~[forge:?]
	at java.util.Optional.map(Unknown Source) ~[?:1.8.0_291]
	at net.minecraftforge.fml.network.NetworkHooks.onCustomPayload(NetworkHooks.java:91) ~[forge:?]
	at net.minecraft.client.network.play.ClientPlayNetHandler.func_147240_a(ClientPlayNetHandler.java:1904) ~[?:?]
	at net.minecraft.network.play.server.SCustomPayloadPlayPacket.func_148833_a(SCustomPayloadPlayPacket.java:59) ~[?:?]
	at net.minecraft.network.play.server.SCustomPayloadPlayPacket.func_148833_a(SCustomPayloadPlayPacket.java:11) ~[?:?]
	at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?]
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(SourceFile:103) ~[?:?]
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:947) ~[?:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) ~[?:?]
	at net.minecraft.client.main.Main.main(Main.java:184) ~[1.16.5.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_291]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_291]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_291]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_291]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.16.5-36.2.2.jar:36.2]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.9.jar:?]
```

log: 
[latest.log](https://github.com/Alex-the-666/Ice_and_Fire/files/6986726/latest.log)

**Comment by Red6785** [comment]
Downgrade citadel from 1.7.3 to 1.7.1

**Comment by Blunder-G** [comment]
> Downgrade citadel from 1.7.3 to 1.7.1

Strange. It worked but in this case Citadel 1.7.3 has a critical problem.

**Comment by AlexModGuy** [comment]
fixed in 2.1.9
